src/__tests__/wingman.test.ts

```diff
@@ -75,13 +79,28 @@ describe("WingmanClient", () => {
 
       expect(exec).toHaveBeenCalledExactlyOnceWith(path, [], {
         env: expect.objectContaining({
-          LLM_SERVER_URL: getFlyCIUrl(),
+          LLM_SERVER_URL: getWingmanUrl(),
           LLM_API_KEY: accessToken,
           FLYCI_WINGMAN_OUTPUT_FILE: p.join(tmpPath, "wingman.json"),
         }),
       });
     });
 
+    it("when wingman url input is set should use it", async () => {
+      process.env[wingmanUrlEnv] = "mock-url";
+      mockExec.mockResolvedValueOnce(0);
```

Comment:
I will go with mocking `core.getInput()` instead of relying on its internal implementation.

Reply:
What I don't like about mocking is that this would also mock `getInput` for the "flyci-url" which can potentially make the test not that reliable unless we use `mockImplementation` and check if the input is "wingman-url". Otherwise the test will pass even if we use `getFlyciUrl` instead of `getWingmanUrl`

Reply:
opted for verifying of `getInput` was called with `wingman-url` the second time